Spec/Auth.swift

```diff
@@ -109,20 +109,19 @@ override class var defaultTestSuite : XCTestSuite {
             static let ttl = 1.0
             static let capability = "{\"cansubscribe:*\":[\"subscribe\"]}"
         }
-    override func spec() {
 
-        describe("Basic") {
+        
 
             // RSA1
-            it("should work over HTTPS only") {
+            func test__003__Basic__should_work_over_HTTPS_only() {
```

Comment:
The number in the test name (`test__003__`) ensures that XCTest will run the tests in the same order as Quick would have run them. We have some tests which will fail if they are not run in the same order as they are in our Quick test suite.

Reply:
That's bad, I've created Issue for this https://github.com/ably/ably-cocoa/issues/1243